contributing/diversity/further_reading.rst

```diff
@@ -0,0 +1,56 @@
+Further reading / viewing
+====================
```

Comment:
Please match the length of the underline, the length of the headline.

Same for the others, thanks